app/models/miq_request_workflow.rb

```diff
@@ -192,7 +192,17 @@ def validate(values)
         if fld[:required] == true
           # If :required_method is defined let it determine if the field is value
           unless fld[:required_method].nil?
-            fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            # Check to see if we have a list of validation methods
+            if fld[:required_method].kind_of? Enumerable
+              fld[:required_method].each do |method|
+                fld[:error] = send(method, f, values, dlg, fld, value)
+                # Bail out early if we see an error
+                break unless fld[:error].nil?
+              end
+            else
+              fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            end
+
```

Comment:
A second check for a gce-specific validate_vm_name would work too. I do tend to favor this approach though, as it follows the more declarative style of putting the field validations and their restrictions in `miq_provision_google_dialogs_template.yaml`. 

However, if I truly believed that, I'd change `:validate_vm_name` to the more direct `validate_length` as that's much more declarative. Another downside is that it might be surprising for `fld[:required_method]` to be optionally enumerable.  I could go either way on this -  what do you think?